flink-test-utils-parent/flink-test-utils/pom.xml

```diff
@@ -163,6 +163,17 @@ under the License.
 			<scope>runtime</scope>
 		</dependency>
 
+		<!-- This dependency is added for `flink-benchmarks` project which will
+		use zookeeper testing server to benchmark olap performance. This dependency
+		is added here to ensure that its version is consistent with the flink
+		version in the `flink-benchmarks` and to avoid conflicts. Removing this
+		dependency may cause failure of `flink-benchmarks`. -->
+		<dependency>
+			<groupId>org.apache.curator</groupId>
+			<artifactId>curator-test</artifactId>
+			<version>${curator.version}</version>
+		</dependency>
+
```

Comment:
As discussed in https://github.com/apache/flink-benchmarks/pull/78, 
> We might want to add a comment to the corresponding line in [apache/flink:pom.xml:144](https://github.com/apache/flink/blob/ec6ebe2d22d15883f7236895387a45a533cfefe0/pom.xml#L144) to point out that updating the curator version in Flink should result in updating the corresponding version in Flink's benchmark tests as well.

Reply:
We don't need to add corator-test dependency in flink-benchmark after adding it here, so we may not need to add comment in [apache/flink:pom.xml:144](https://github.com/apache/flink/blob/ec6ebe2d22d15883f7236895387a45a533cfefe0/pom.xml#L144). WDYT?